pkg/security/tests/security_profile_test.go

```diff
@@ -800,6 +806,8 @@ func TestSecurityProfileAutoSuppression(t *testing.T) {
 		enableSecurityProfile:                   true,
 		securityProfileDir:                      outputDir,
 		securityProfileWatchDir:                 true,
+		enableAutoSuppression:                   true,
+		autoSuppressionEventTypes:               []string{"exec", "dns"},
```

Comment:
Now that anomaly detection events and auto-suppression events are decouple, we should test that they really are.
At least, running this tests by specifying an empty list of anomaly detection event types (by default they are all enabled); and doing the opposite (enabling all anomaly detection events, disabling all auto-suppression events, ensuring that they are no more suppressed).

Same applies for anomaly detection tests.

Reply:
Could be done in a separate PR :)